packages/api-key/integration-tests/__tests__/api-key-module-service.spec.ts

```diff
@@ -180,8 +178,7 @@ moduleIntegrationTestRunner({
         it("should not reflect any updates on other fields", async function () {
           const createdApiKey = await service.create(createSecretKeyFixture)
 
-          const updatedApiKey = await service.update({
-            id: createdApiKey.id,
+          const updatedApiKey = await service.update(createdApiKey.id, {
```

Comment:
q: why did we move the id here?

Reply:
@adrien2p I just followed the API pattern of the region, as I assumed that's how we want all modules to behave